.rubocop_todo.yml

```diff
@@ -33,12 +33,12 @@ Metrics/LineLength:
 # Offense count: 32
 # Configuration parameters: CountComments.
 Metrics/MethodLength:
-  Max: 54
+  Max: 55
 
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 314
+  Max: 315
```

Comment:
Those changes are getting a little bit annoying. We're basically incrementing each time and making those rules irrelevant. Wanted to flag as we likely should disable the rule for the 2 specific parts of code we care about.

Reply:
Yeah, we should just kill these. I can do it.